tox.ini

```diff
@@ -15,7 +15,7 @@ deps =
   isort
 
 commands =
-  flake8 --verbose {env:CI_FLAGS:} --exclude="packages/*/src,packages/*/result,packages/cache,.git" ./
+  flake8 --verbose --exclude="packages/*/src,packages/*/result,packages/cache,.git" gen packages pkgpanda pytest release ssh test_util
```

Comment:
why does the " do all the files" not work? I really would rather not have people manually adding things.

Also, the CI_FLAGS stuff makes teamcity / ci understand the output better.